generate_nibetaseries_images.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+set -e
+
+# Generate Dockerfile.
+generate_docker() {
+  docker run --rm kaczmarj/neurodocker:0.5.0 generate docker \
+    --base=neurodebian:stretch-non-free \
+    --install gcc g++ graphviz \
+    --pkg-manager=apt \
+    --user=neuro \
+    --miniconda \
+      conda_install="python=3.6" \
+      pip_install="jupyter seaborn tox" \
+      create_env="neuro_py36" \
+      activate=true \
+    --copy . /src/nibetaseries \
+    --user=root \
+    --run 'chown -R neuro:users /src/nibetaseries' \
+    --user=neuro \
+    --workdir /src/nibetaseries \
+    --run 'pip install --no-cache-dir .' \
+    --run 'mkdir -p ~/.jupyter && echo c.NotebookApp.ip = \"0.0.0.0\" > ~/.jupyter/jupyter_notebook_config.py' \
+    --workdir /home/neuro
+
+}
+
+generate_docker > Dockerfile
+
+docker build -t hbclab/nibetaseries:dev .
+# singularity build hbclab_nibetaseries.simg Singularity
```

Comment:
I removed singularity since I was having trouble building the image and I think it would be alright for right now to use the image on dockerhub to build singularity containers (similar to how I use fmriprep).

Reply:
True that, might/will be easier after all. Should I include the respective information on installation, setup and running in the [README](https://github.com/HBClab/NiBetaSeries/blob/master/README.rst)?

Reply:
If you could add that information in those locations, that would be great, thanks!

Reply:
I'm on it and will do the PR tomorrow morning!

Reply:
Quick question regarding the installation instructions: should only the installation itself be covered or also some pointers on how to run it (e.g., `docker/singularity run ....`)?

Reply:
I'm partial to how [fmriprep](https://fmriprep.readthedocs.io/en/stable/installation.html) organized their installation page, so a quick example on what the command could look like is useful.

here is an example template of the command:
```
nibs -c WhiteMatter CSF \
--participant_label 001 \
-w {work_dir} \
-a {atlas_mni_file} \
-l {atlas_tsv} \
{bids_dir} \
fmriprep \
{out_dir} \
participant
```

Reply:
Just submitted my initial version [here](https://github.com/HBClab/NiBetaSeries/pull/140/commits/fc6a2a9c0b58b49aa451e437194486a0f6670040). It's heavily based on the respective sections of the [fmriprep](https://fmriprep.readthedocs.io/en/stable/installation.html) and [mriqc](https://mriqc.readthedocs.io/en/stable/install.html) docs. However, I wasn't sure about the exact wording, as well as if you would prefer a mapping of all arguments individually (e.g., `work_dir`, `atlas_mni_file`, etc.) during the initiation of the containerized versions. Please feel free to change these things based on your preference.

Reply:
@jdkent would this version work for you? If so, I think we can merge.